docs/src/examples/components/Form/Types/FormExample.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { Form, Input, Button } from '@stardust-ui/react'
+
+const FormExample = () => (
+  <Form
+    onSubmit={() => {
+      alert('Form submitted')
+    }}
+  >
+    <Form.Field
+      label="First name"
+      controlType={Input}
+      name="firstName"
+      id="first-name"
+      required={true}
+    />
+    <Form.Field
+      label="Last name"
+      controlType={Input}
+      name="lastName"
+      id="last-name"
+      required={true}
+    />
+    <Form.Field
+      label="I agree to the Terms and Conditions"
+      controlType="input"
+      type="checkbox"
+      id="conditions"
+    />
+    <Form.Field controlType={Button} control={{ content: 'Submit' }} />
```

Comment:
proceeding with the thought mentioned above - maybe this could be considered as an alternative for the first stages?
```jsx
<Form.Field control={{ as: Button, content: 'Submit' }} />
```

Reply:
Please take a look into the previous comment: https://github.com/stardust-ui/react/pull/353#discussion_r226324506